frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/OverrideForkPointBackgroundable.java

```diff
@@ -88,6 +87,11 @@ private void setOverrideForkPointConfigValues(
     }
 
     try {
+      // As a step towards deprecation of `whileDescendantOf` key (see #1580),
```

Comment:
This was already in place for some time, actually (piggybacked with some unrelated changes).